framework/web/filters/CHttpCacheFilter.php

```diff
@@ -193,7 +193,7 @@ protected function sendCacheControlHeader()
 	{
 		if(Yii::app()->session->isStarted)
 		{
-			session_cache_limiter('public');
+			Yii::app()->session->setCacheLimiter('public');
```

Comment:
What is purpose of `session_cache_limiter()` here? This change broken `CHttpCacheFilter` in my case - for some reason I get `last-modified` header always set to mtime of entry script (which is obviously incorrect), and I'm unable to get 304 response from this filter.

Reply:
These should be equal.

Reply:
Disabling and reenabling session is definitely not transparent - I was also wondering why each requests sets session cookie, and this was the reason :P.